### PR TITLE
update ci build

### PR DESCRIPTION
**Pull request by mr-tz**
current CI build fails, see build Action

```
Fatal Python error: init_fs_encoding: failed to get the Python codec of the filesystem encoding
Python runtime state: core initialized
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007f06624a3700 (most recent call first):
<no Python frame>
```

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
```
Comment (mr-tz) [inline .github/dependabot.yml:1]: unrelated addition

**Comment by mr-tz** [comment]
Who can make sense of this? All my research lead nowhere. Build and spec are almost identical to `capa` which works fine.

**Comment by williballenthin** [comment]
https://github.com/pyinstaller/pyinstaller/issues/5438

**Comment by williballenthin** [comment]
image

the `encodings` module does seem to be present i8n `base_library.zip`.

**Comment by williballenthin** [comment]
see #371 that duplicates and extends this PR

**Pull request closed without merging**